src/core_plugins/metrics/public/kbn_vis_types/request_handler.js

```diff
@@ -9,11 +11,20 @@ const MetricsRequestHandlerProvider = function (Private, Notifier, config, timef
     name: 'metrics',
     handler: function (vis /*, appState, uiState, queryFilter*/) {
 
+      let timezone = config.get('dateFormat:tz', 'Browser');
```

Comment:
Yeah... but we need to move it out of Timelion and put it somewhere more global. Suggestions?